initfiles/componentfiles/thor/run_thor

```diff
@@ -35,7 +34,7 @@ echo $$ > $RUN_THOR_PID_NAME
 export SENTINEL="thor.sentinel"
 while [ 1 ]; do
     # update slaves file in case state of environment has been altered since last run
-    daliadmin $DALISERVER dfsgroup ${groupName} slaves
+    daliadmin server=$DALISERVER dfsgroup ${groupName} slaves
```

Comment:
This change was made because I've been researching using dns instead of environments defined by ip's.  If we're using ip's then it works without using server=.  If we instead use a domain name, daliadmin will parse the domain name as though it was a command instead of an address.  server= just makes it more robust so in the future we can use either.  It's sort of a side issue that I've been noticing when playing around, so I took the chance to make the change here.